config/watcher_local_test/kustomization.yaml

```
@@ -23,6 +23,7 @@ patches:
     - op: add
       path: /spec/template/spec/containers/0/args/-
       value: --enable-watcher-local-testing
+      value: --in-kcp-mode=true
```

```suggestion
    - op: add
      path: /spec/template/spec/containers/0/args/-
      value: --in-kcp-mode=true
```

Lets stick to the normal yaml pattern and avoid having duplicate keys, otherwise we will get the warning `Key 'value' is duplicated `.

If you would like to change it to use `value` multiple times in one `op`, then I would suggest to change to everywhere. But your IDE will flood you with warnings 😓